website/cue/reference/remap/functions/seahash.cue

```diff
@@ -0,0 +1,37 @@
+package metadata
+
+remap: functions: seahash: {
+	category:    "Cryptography"
+	description: """
+		Calculates a [Seahash](\(urls.seahash)) hash of the `value`.
+		Note: function converts unsigned int-64 seahash to signed int-64 to met vrl standards, so integer overflow happens when calculated hash is higher than signed int-64 max value. Depending on the use-case this might or mightn't be an issue.
+		"""
+
+	arguments: [
+		{
+			name:        "value"
+			description: "The string to calculate the hash for."
+			required:    true
+			type: ["string"]
+		},
+	]
+	internal_failure_reasons: []
+	return: types: ["string"]
```

Comment:
I think this should be where that `integer` from the `sha1` docs belongs.

Reply:
Yes exactly. Thanks for noticing.